release-notes.md

```diff
@@ -310,6 +318,68 @@
     - [Unit testing](#unit-testing)
     - [Integration tests parallelization](#integration-tests-parallelization)
 
+# v37.0
+
+This release has been tracked in our [roadmap
+project](https://github.com/orgs/cloud-hypervisor/projects/6) as iteration
+v37.0. The following user visible changes have been made:
+
+### Long Term Support (LTS) Release
+
+This release is a LTS release. Point releases for bug fixes will be made
+for the next 18 months; live migration and live upgrade will be
+supported between the point releases of the LTS.
+
+### Multiple PCI segments Support for 32-bit VFIO devices
+
+Now VFIO devices with 32-bit memory BARs can be attached to non-zero PCI
+segments on the guest, allowing users to have more 32-bit devices and
+assign such devices to appropriate NUMA nodes for better performance.
```

Comment:
@rbradford PTAL. When I started to rewrite this section and realize the discussion we had about the practical number of 32-bit devices we can use given the very limited 32-bit address space. (See https://github.com/cloud-hypervisor/cloud-hypervisor/pull/5953#issuecomment-1812916352).

For this reason I didn't mention anything specific about how many 32-bit devices we can use and keep it very general, e.g. `allowing users to have more 32-bit devices`. Please let me know if you have suggestions to improve it. Thanks.

Reply:
Sounds great to me!

Reply:
Great. Land this PR and cut the release now.